firestore/query.go

```diff
@@ -85,7 +85,7 @@ type PlanSummary struct {
 	// The indexes selected for the query. For example:
 	//
 	//	[
-	//	  {"query_scope": "Collection", "properties": "(foo ASC, __name__ ASC)"},
+	//	  {"query_scope": "Collection", "properties": "[foo ASC, __name__ ASC)"},
```

Comment:
change close paren to close square bracket.
Still not sure what this is. A map[string]any where the first value is a string and the second a []string?
It would be clearer to write the Go syntax for that.

Reply:
This is the format backend returns
It does not return array

Reply:
So it's a `[]map[string]any`. Let's at least document that.
It one of your tests, add `fmt.Printf("type=%T\n", IndexesUsed[0]["properties"])` so we can figure out the type of "properties". It might be worth documenting that too.

Reply:
Added